### DISCUSSION TITLE
StdScheduler中ResumeJob和PauseJob的原子性操作

### DISCUSSION THREAD

**Discussion by miJasonK** [reugn/go-quartz#127] [Show and tell]
在StdScheduler对象中，如果Remove Job成功但Push Job失败，这样会产生任务丢失，此场景应该怎么去支持？
In the StdScheduler object, if the Remove Job succeeds but the Push Job fails, the task will be lost. How should this scenario be supported?

**Comment by reugn** [comment]
See #128.
